ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/Dialog_DataInput.java

```diff
@@ -22,18 +24,23 @@
 public class Dialog_DataInput extends  DialogFragment {
 
     RadioGroup radioSexGroup;
-    EditText et_age;
-    EditText et_rest_hr;
+    ValidatableEditText et_age;
```

Comment:
Maybe more suitable name will be NumericEditText ... proper naming is getting important once you have more classes in your project.

Reply:
done